share.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Share
+excerpt: "Do you like ev3dev? Show your support! We invite you to star the repo on GitHub and submit your cool projects to our projects page."
```

Comment:
Ummm... But that's not what this page does. Are you saying that that's what it _should_ be?